src/Jackett.Common/Definitions/divteam.yml

```diff
@@ -125,16 +125,16 @@ search:
       selector: a[href^="index.php?page=torrent-details"][onmouseover]:not(:contains("VOSE"))
       optional: true
       filters:
-        - name: re_replace
-          args: ["\\W", "."] # Spaces and other characters -> .
         - name: append
-          args: ".MULTi/Spanish" # DivTeam doesn't bring languages. Usually Original + Spanish.
+          args: " MULTi/SPANiSH" # DivTeam doesn't bring languages. Usually Original + Spanish.
         - name: re_replace
           args: ["\\.+", "."] # More than 1 dot -> .
         - name: re_replace
           args: ["^\\.", ""] # Delete first dot
-        - name: re_replace
+        - name: replace
           args: ["UHDRip", "BDRip"] # Fix for Radarr
```

Comment:
**If** these are 4K results as well:
```suggestion
          args: ["UHDRip", "Bluray-2160p"] # Fix for Radarr
```

Reply:
Sorry. Don't see the suggestion, so I dod the change myself. 😅